pkgs/by-name/c2/c2000-cgt/package.nix

```diff
@@ -0,0 +1,55 @@
+{
+  buildFHSEnv,
+  lib,
+  stdenvNoCC,
+}:
+
+let
+  fhs = buildFHSEnv {
+    name = "fhs-shell";
+  };
+
+  package = stdenvNoCC.mkDerivation rec {
+    pname = "cgt_c2000";
+    version = "22.6.1.LTS";
+
+    filename = "ti_${pname}_${version}_linux-x64_installer.bin";
+
+    src = builtins.fetchurl {
+      url = "https://dr-download.ti.com/software-development/ide-configuration-compiler-or-debugger/MD-xqxJ05PLfM/${version}/${filename}";
+      sha256 = "sha256:1nikmcgja2i22qk4alclfqiaqxwc7k0m9hwl5pviiv43bivb3rw7";
+    };
+
+    unpackPhase = "true";
+
+    installPhase = ''
+      runHook preInstall
+
+      mkdir -p $out/install/
+      cp -a $src $out/install/installer.bin
+
+      chmod +x $out/install/installer.bin
+      ${fhs}/bin/fhs-shell -c '$out/install/installer.bin --mode unattended --prefix /build/ti'
```

Comment:
I think you can just set the prefix to `$out` and then move everything outside the `ti-cgt-c2000_${version}` directory